.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+      - '[0-9]+.x'
+      - '[0-9]+.[0-9]+.x'
+jobs:
+  release:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    timeout-minutes: 2
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          persist-credentials: false
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+        # additional validation steps can be run here
+      - name: Semantic Release
+        uses: BrightspaceUI/actions/semantic-release@master
+        with:
+          GITHUB_TOKEN: ${{ secrets.D2L_GITHUB_TOKEN }}
+          NPM: true
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          DRY_RUN: true
```

Comment:
This reminds me... did we remove this from `d2l-fetch`?

Reply:
not yet. The first run it determined there was no change, which was all i wanted to test.